crates/sui-indexer-alt/src/handlers/sum_coin_balances.rs

```diff
@@ -99,9 +99,17 @@ impl Processor for SumCoinBalances {
                     continue;
                 };
 
-                // Coin balance only tracks address-owned objects
-                let Owner::AddressOwner(owner_id) = object.owner() else {
-                    continue;
+                // Coin balance only tracks address-owned or ConsensusV2 objects
+                let (owner_kind, owner_id) = match object.owner() {
+                    Owner::AddressOwner(owner_id) => (StoredOwnerKind::Address, owner_id),
+                    // ConsensusV2 objects are treated as address-owned for now in indexers.
+                    // This will need to be updated if additional Authenticators are added.
+                    Owner::ConsensusV2 { authenticator, .. } => (
+                        StoredOwnerKind::ConsensusV2,
+                        authenticator.as_single_owner(),
```

Comment:
Just noting my momentary confusion that this doesn't return an `Option<...>`!

Reply:
yes, i put a comment on the function to clarify in another PR